benchmarks/torchbench_model.py

```diff
@@ -306,10 +306,15 @@ def load_benchmark(self):
     # workaround "RuntimeError: not allowed to set torch.backends.cudnn flags"
     # torch.backends.__allow_nonbracketed_mutation_flag = True
 
-    # torchbench uses `xla` as device instead of `tpu`
-    device = (
-        str(self.benchmark_experiment.get_device())
-        if self.is_accelerator_tpu() else self.benchmark_experiment.accelerator)
+    if self.should_initialize_on_xla():
```

Comment:
Can you remind me why we didn't initialize on CUDA device for GPU experiment before? Thanks

Reply:
We actually do initialize them on CUDA device (i.e. `self.benchmark_experiment.accelerator`). The thing is that for `moco` we have to initialize the benchmark on XLA device (i.e. `str(self.benchmark_experiment.get_device())`).